Documentation/Blazorise.Docs/Pages/Blog/2024-07-19_HowToStyleBlazoriseComponents/Index.md

```diff
@@ -0,0 +1,162 @@
+
+### CSS classes
+
+Applying CSS classes is as straight forward as plain on Blazorise, we just specify the class like so:
+
+```html|BlazoriseCSSExample
+<Alert Color="Color.Success" Visible>
+    <AlertMessage>Well done!</AlertMessage>
+    <AlertDescription Class="bg-topo">You successfully read this important alert message.</AlertDescription>
+</Alert>
+```
+
+### Theming
+
+Blazorise has support for themes. Customize Blazorise with your theme. You can change the colors, the typography and much more using themes.
+
+To learn more about theming, head over to the [documentation page](https://blazorise.com/docs/theming)
+
+## Limitations of CSS Isolation
+
+Blazorise just like other Blazor frameworks has a limitation with CSS isolation, you see, CSS Isolation works by compiling the CSS styles and bundling them up with the Assembly.
+
+CSS isolation occurs at build time. Blazor rewrites CSS selectors to match markup rendered by the component. The rewritten CSS styles are bundled and produced as a static asset. The stylesheet is referenced inside the `<head>` tag ([location of `<head>` content](https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#location-of-head-and-body-content))
+
+The following `<link>` element is added by default to an app created from the Blazor project templates:
+
+```html|BlazorHtmlStylesLink
+<link href="{ASSEMBLY NAME}.styles.css" rel="stylesheet">
+```
+
+The `{ASSEMBLY NAME}` placeholder is the project's assembly name.
+
+Within the bundled file, each component is associated with a scope identifier. For each styled component, an HTML attribute is appended with the format `b-{STRING}`, where the `{STRING}` placeholder is a ten-character string generated by the framework. The identifier is unique for each app.
+
+The problem here is that, the bundle is generated at compile time of the CSS.
+
+### Let me illustrate this limitation with an example.
+
+Let's create a component in our project, call it `TestComponent`
+
+`TestComponent.razor`
+```html|TestComponentRazor
+<Div Class="isolated-class-name">
+    ...
+</Div>
+```
+
+Then create the scoped CSS file for it
+
+`TestComponent.razor.css`
+```css|TestComponentRazorCss
+.isolated-class-name {
+    /* class info here */
+}
+```
+
+This will not work, because when your application is compiled, the generated `b-{STRING}` will be different than that of Blazorise's. This is a known limitation of Blazor, but the workaround is very simple!
+
+### The workaround:
+This is not really a "workaround" rather a different approach at the issue. You cannot make 3rd party libraries work with your isolated CSS classes, **however** what you can do, is move those classes to a separate CSS file hosted under `wwwroot`.
+
+# Thanks for reading this blog and using Blazorise!!!
```

Comment:
We have a lax attitude in general, but this seems too informal/unprofessional?
I'd not have this last paragraph or at least word it differently, like "We expect you in the next blog post" or something like that.
Also making it h1 might interfere with SEO, and it's not relevant to the content at all.

Reply:
Alright, I will fix it!